docs/src/man/comparisons.md

```diff
@@ -1,77 +1,204 @@
 # Comparisons
 
-This section compares DataFrames.jl with other data manipulation frameworks.
+This section compares DataFrames.jl with other data manipulation frameworks in Python, R, and Stata.
+
+A sample data set can be created using the following code:
+
+```julia
+df = DataFrame(id = 'a':'f', grp = repeat(1:2, 3), x = 6:-1:1, y = 4:9, z = [3:7; missing])
+df2 = DataFrame(grp = 1, w = 10)
+```
+
+## Comparison with the Python package pandas
+
+The following table compares the main functions of DataFrames.jl with the Python package pandas (version 1.1.0):
+
+```python
+df = pandas.DataFrame({'grp': [1, 2, 1, 2, 1, 2],
+                       'x': range(6, 0, -1),
+                       'y': range(4, 10),
+                       'z': [3, 4, 5, 6, 7, None]},
+                      index = list('abcdef'))
+
+>>> df
+   grp  x  y    z
+a    1  6  4  3.0
+b    2  5  5  4.0
+c    1  4  6  5.0
+d    2  3  7  6.0
+e    1  2  8  7.0
+f    2  1  9  NaN
+```
+
+By comparison, this pandas data frame has `a` to `f` as row indices rather than a separate `id` column.
+
+### Accessing data
+
+| Operation                  | pandas                 | DataFrames.jl                      |
+| :------------------------- | :--------------------- | :--------------------------------- |
+| Cell indexing by location  | `df.iloc[1, 1]`        | `df[2, 2]`                         |
+| Row slicing by location    | `df.iloc[1:3]`         | `df[2:3, :]`                       |
+| Column slicing by location | `df.iloc[:, 1:]`       | `df[:, 2:end]`                     |
+| Row indexing by label      | `df.loc['c']`          | `df[findfirst(==('c'), df.id), :]` |
+| Column indexing by label   | `df.loc[:, 'x']`       | `df[:, :x]`                        |
+| Column slicing by label    | `df.loc[:, ['x','z']]` | `df[:, [:x, :z]]`                  |
+|                            | `df.loc[:, 'x':'z']`   | `df[:, Between(:x, :z)]`           |
+| Mixed indexing             | `df.loc['c'][1]`       | `df[findfirst(==('c'), df.id), 2]` |
+
+Note that Julia uses 1-based indexing, inclusive on both ends. A special keyword `end` can be used to
+indicate the last index.
+
+In the DataFrames.jl examples, the `findfirst` function is used to find the first match and return the result
+as a single `DataFrameRow` object. In the case that `id` is not unique, you can use the `findall` function
+or boolean indexing instead. It would then return a `DataFrame` object containing all matched rows. The following
+two lines of code are functionally equivalent:
+
+```julia
+df[findfirst(==('c'), df.id), :]
+df[df.id .== 'c', :]
+```
+
+### Common operations
```

Comment:
I would add this sentence, as this is a point of the discussion why `findfirst` and `findall` are distinct:
```suggestion
Note that this differs from `df.loc['c']` behaviour in pandas which
returns a `Series` object if there is exactly one `'c'` value in the index,
and a `DataFrame` object if there are multiple occurrences of `'c'` in it.

### Common operations
```

Reply:
Done. See latest commit.